libraries/classes/InsertEdit.php

```diff
@@ -718,12 +706,12 @@ private function getHtmlInput(
     /**
      * Get HTML select option for upload
      *
-     * @param string $vkey   [multi_edit]['row_id']
-     * @param array  $column description of column in given table
+     * @param string $vkey      [multi_edit]['row_id']
+     * @param string $field_MD5 array index as an MD5 to avoid having special characters
```

Comment:
```suggestion
     * @param string $fieldHashMd5 array index as an MD5 to avoid having special characters
```
This is maybe better for our coding standard